person.rb

```diff
@@ -0,0 +1,30 @@
+require 'nameable'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    super

```

Comment:
* Kindly add `super( )` as I get an argument error as `nameable is not expecting any argument [Kindly read more on the super keyword](https://www.rubyguides.com/2018/09/ruby-super-keyword/) 
![image](https://user-images.githubusercontent.com/52497246/213361057-e4885f75-cc02-4723-bac9-2f002283aa33.png)